spec/print_the_board_spec.rb

```diff
@@ -1,15 +1,23 @@
-require 'tic_tac_toe'
-
-RSpec.describe TicTacToe::Board
+require 'print_the_board'
+#testing
+RSpec.describe TicTacToe::DisplayBoard
     context "it should print out an array for the new gameboard" do
         it "should print out a gameboard" do
-            board = TicTacToe::Board.new
+            # given
+            fakey_board = [ [ "-", "-", "-" ], 
+                            [ "-", "-", "-" ],
+                            [ "-", "-", "-" ] ]
+            board = TicTacToe::DisplayBoard.new(fakey_board)
+
+            # when
+            actual = board.to_s
+
+            # then
             expected = "| - | - | - |\n"\
-                       "-------------\n"\
+                       "+---+---+---+\n"\
                        "| - | - | - |\n"\
-                       "-------------\n"\
+                       "+---+---+---+\n"\
                        "| - | - | - |"
-                       
-            expect(board.to_s).to eq(expected)
+            expect(actual).to eq(expected)
     end
 end
```

Comment:
Just because I hate red - let's make sure each file ends in a new line. Small detail, should be its own commit, since its stupid. But yeah!